resolvers/webpack/index.js

```diff
@@ -311,6 +311,22 @@ function getResolveSync(configPath, webpackConfig, cwd) {
   return cached.value;
 }
 
+const _evalCache = new Map();
+function evaluateFunctionConfigCached(configPath, webpackConfig, env, argv) {
+  const cacheKey = JSON.stringify({ configPath, args: [env, argv] });
```

Comment:
This key is key ordering sensitive in contrast to `hashify()`. https://github.com/import-js/eslint-plugin-import/blob/67cc79841fc823ad4af2532af2dc6704e4b3b03a/utils/hash.js#L15

I consider should be fine because:
- In most practical cases, key ordering will be stable.
- Even when ordering is not stable, it will just increase cache miss. Just get almost as fast as `cache: false` at worst. And won't cause invalid result.
- Even if it cause unexpected problems, existing users won't have problem immediately since cache feature is opt-in.